examples/timestamping/backend/src/api.rs

```diff
@@ -84,6 +87,18 @@ impl PublicApi {
         })
     }
 
+    /// Endpoint for getting service configuration.
+    pub fn get_service_configuration(self, state: &ServiceApiState<'_>) -> api::Result<Config> {
+        let schema = Schema::new(state.service_data());
+        let config_entry = schema.config.get();
+
+        if let Some(config) = config_entry {
```

Comment:
Nit: You can use [`Option::ok_or_else`](https://doc.rust-lang.org/std/option/enum.Option.html#method.ok_or_else).